consensus/consensus_test.go

```diff
@@ -95,6 +96,29 @@ func newTestConsensus() (*testConsensus, error) {
 		return nil, err
 	}
 
+	addr := thor.BytesToAddress([]byte("to"))
+	cla := tx.NewClause(&addr).WithValue(big.NewInt(10000))
+	transaction := new(tx.Builder).
+		ChainTag(repo.ChainTag()).
+		GasPriceCoef(1).
+		Expiration(10).
+		Gas(21000).
+		Nonce(1).
+		Clause(cla).
+		BlockRef(tx.NewBlockRef(0)).
+		Build()
+
+	sig, err := crypto.Sign(transaction.SigningHash().Bytes(), genesis.DevAccounts()[0].PrivateKey)
+	if err != nil {
+		return nil, err
+	}
+	transaction = transaction.WithSignature(sig)
```

Comment:
You can reuse the txBuilder; the addr won't be needed since it's already added by the function and the test is passing
```suggestion
	txBuilder := txBuilder(repo.ChainTag()).Clause(cla)
	transaction := txSign(txBuilder)
```